### PR TITLE
Make some things a little bit clearer

### PR DESCRIPTION
**Pull request by volkflo**
*Description of changes:*
Some little rewording to make some things a bit more clear in the docs.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/install/docker-security.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ nav_order: 7
 
 Before deploying to a production environment, you should replace the demo security certificates and configuration YAML files with your own. With the RPM and Debian installations, you have direct access to the file system, but the Docker image requires modifying the Docker Compose file to include the replacement files.
 
+Additionally you may set the docker environment variable `DISABLE_INSTALL_DEMO_CONFIG` to `true`. This will completely disable all changes that are made by the demo installer.
```
Comment (bekihm) [inline docs/install/docker-security.md:12]: Maybe we can write here also the benefits to set it true?

File: docs/security/configuration/security-admin.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ opendistro_security.authcz.admin_dn:
   - CN=kirk,OU=client,O=client,L=test,C=DE
 ```
 
-Do not use node certificates as admin certificates. The best practice is to keep these certificates separate. Also, do not use any whitespace between the parts of the DN.
```
Comment (aetter) [inline docs/security/configuration/security-admin.md:27]: Hi @volkflo, is this true? My understanding is that you can technically do it, but that it's a really bad idea.

Comment (volkflo) [reply docs/security/configuration/security-admin.md:27]: Hi @aetter,
Thanks for the fast feedback. 
I can show you my thoughts about that:

`elasticsearch.yml`
```
...
  authcz:
    admin_dn:
     - "CN=docker-node,O=Scheer,L=Saarbruecken,ST=Saarland,C=DE"
  nodes_dn:
    - "CN=docker-node,O=Scheer,L=Saarbruecken,ST=Saarland,C=DE"
...
```
If I run the `securityadmin.sh` script it prints that output:

> [root@mynode elasticsearch]# chmod +x /usr/share/elasticsearch/plugins/opendistro_security/tools/securityadmin.sh && /usr/share/elasticsearch/plugins/opendistro_security/tools/securityadmin.sh -icl -nhnv -h my-host -cacert /usr/share/elasticsearch/config/root-ca.pem -cert /usr/share/elasticsearch/config/node.pem -key /usr/share/elasticsearch/config/node-key.pem -cd /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/
Open Distro Security Admin v7
Will connect to my-host:9300 ... done
Connected as CN=docker-node,O=Scheer,L=Saarbruecken,ST=Saarland,C=DE
ERR: Seems you use a node certificate which is also an admin certificate
     That may have worked with older Open Distro Security versions but it indicates
     a configuration error and is therefore forbidden now.

So, recording to that error message it is not possible to use the same certificates.

**Comment by volkflo** [comment]
@aetter Thanks for merging 👍

**Comment by aetter** [comment]
Thanks for contributing!

**Pull request merged**